src/otx/algorithms/common/adapters/mmcv/utils/config_utils.py

```diff
@@ -977,3 +973,25 @@ def area(x):
         input_size = self.select_closest_size(input_size, input_size_preset)
         logger.info(f"-> Closest preset: {input_size}")
         return input_size
+
+
+def get_proper_repeat_times(
```

Comment:
Could you add unit test for this function?

Reply:
I think this test includes this functionality. 
https://github.com/openvinotoolkit/training_extensions/blob/e1cf24fe04c36af2963f098052c248475a68e2b7/tests/unit/algorithms/common/adapters/mmcv/hooks/test_adaptive_repeat_data_hook.py#L33

Reply:
Yes it is in some degree. But it's insufficient for testing various situation, I think. For example, it can be better by having a code dealing with a wrong argument value in this function with those test cases.

Reply:
Added unit test to check the special cases (i.e. train_data_size = 0)